src/Form/atoms/Checkbox.tsx

```diff
@@ -51,11 +51,11 @@ const IconWrapper = styled.div<{color: ISvgIcons['color']}>`
   }
 `;
 
-const Container = styled.label<{visualState?: CheckboxState, disabled?: boolean}>`
+const Container = styled.label<{visualState?: CheckboxState, disabled?: boolean, parentHover?: boolean}>`
```

Comment:
This is a heavy way to handle hover states from parents. What context is this being used in?

The browser behaviour defaults should mean that anything wrapped in the semantically correct way should pass the hover state to the input. If that isn't working it's better to address the problem there first and get the CSS working as intended. Using props for hover should be a last resort.

Reply:
I admit this is a strange fix, but perhaps we can have a conversation on slack about the best way to approach it.